wren-ui/README.md

```diff
@@ -55,6 +69,66 @@ You can start editing the page by modifying `app/page.tsx`. The page auto-update
 
 This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
 
+## Development wren-ui module on local
+There are many modules in Wren AI, to develop wren-ui, you can start other modules(services) via docker-compose.
+In the [Start wren-ui from source code](#Start-wren-ui-from-source-code) section, you've know how to start wren-ui from the source code to develop.
+To start other modules via docker-compose, you can follow the steps below.
+
+Step 1. Prepare you .env file
+In the WrenAI/docker folder, you can find the .env.example file. You can copy this file to .env.local file.
+
+```
+# assume current directory is wren-ui
+cd ../docker
+cp .env.example .env.local
+```
+Step 2. Modify your .env.local file
+You need to change the value of the environment variables in the .env.local file like `LLM_OPENAI_API_KEY`, `EMBEDDER_OPENAI_API_KEY` and the service version(eg. `WREN_AI_SERVICE_VERSION`) before starting.
+
+Step 3. Start the services via docker-compose
+```
+# current directory is WrenAI/docker
+docker-compose -f docker-compose-dev.yaml --env-file .env.example up
+
+# you can add a -d flag to run the services in the background
+docker-compose -f docker-compose-dev.yaml --env-file .env.example up -d
+# then stop the services via
+docker-compose -f docker-compose-dev.yaml --env-file .env.example down
+```
+
+Step 4. Start wren-ui from source code
+refer to [Start wren-ui from source code](#Start-wren-ui-from-source-code) section to start wren-ui from source code.
+
+## FAQ
+### Can I have multiple project at the same time in Wren AI?
+We currently do not support multiple projects in Wren AI. You can only have one project at a time.
+But there is a workaround for this. Since Wren Engine is stateless and we store your semantic model in the database(Sqlite or Postgres), 
+you can switch between projects by switching the database and make sure you deploying after server started.
+
```

Comment:
(NIT) Suggestion:
Add a tip to describe a brief description - the key point is to use the **SQLITE_FILE** variable to change the DB

```markdown
> Tip: Define the `SQLITE_FILE` variable to specify the path to the database you intend to use.
```